frontend/src/main/components/CourseGradeDist/CourseGradeDistTable.js

```diff
@@ -0,0 +1,60 @@
+import OurTable from "../OurTable";
+
+// eslint-disable-next-line no-unused-vars
+export default function CourseGradeDistTable({ gradeData }) {
+  const columns = [
+    {
+      Header: "id",
+      accessor: "id",
+    },
+    {
+      Header: "Session",
+      accessor: "session",
+    },
+    {
+      Header: "Course",
+      accessor: "course",
+    },
+    {
+      Header: "Instructor",
+      accessor: "instructor",
+    },
+    {
+      Header: "Grade",
+      accessor: "grade",
+    },
+    {
+      Header: "Count",
+      accessor: "count",
+    },
+  ];
+
+  for (const gradeDist of gradeData) {
+    const year = gradeDist.yyyyq.slice(0, 4);
+    let quarter;
+    switch (gradeDist.yyyyq[4]) {
+      case "1":
+        quarter = "Winter";
+        break;
+      case "2":
+        quarter = "Spring";
+        break;
+      case "3":
+        quarter = "Summer";
+        break;
+      default:
+        // There should never be a case where its not 1,2,3,4, right..
+        quarter = "Fall";
+    }
```

Comment:
So something like what you want here already exists in `frontend/src/main/utils/quarterUtilities.js`, though it converts to upper case.

That is you can just write: 
```
import {qtrNumToQuarter} from "main/utils/quarterUtils.js";
```

and then write:

```
quarter = quarterNumToQtr(parseInt(quartergradeDist.yyyyq[4]))
```

replacing 15 lines of code (though this would give you the quarters in all uppercase).

Better yet, add a function to `frontend/src/main/utils/quarterUtilities.js` where you can pass in the yyyyq value and get back exactly what you want, i.e.

```javascript

const qtrNumToQuarterTitleCase = {
  1: "Winter",
  2: "Spring",
  3: "Summer,
  4: "Fall,
};

function yyyyqToQuarterYearTitleCase = (yyyyq) => {
   const year = yyyyq.slice(0, 4);
   const qtrNum = parseInt(yyyyq[4]);
   const quarter = quarterNumToQtrTitleCase(qtrNum);
   return quarter + " " + year;
}
  
Then, you can consder using a map rather than a for loop here.  Something like this:

```
const gradeDataWithSessions = gradeData.map(  g => ({...g, session:  yyyyqToQuarterYearTitleCase(g.yyyyq)}) );
```

Then you'd pass gradeDataWithSessions to `OurTable` instead of `gradeData`.

Over 20 lines of code becomes 1 line of code in this file, and there's a new utility (6 lines of code) that can be reused, along with a 6 line easy to understand constant.

The thing to do: if you are doing a simple thing and its taking a lot of lines of code to do it, and especially if it's a simple data transformation that probably occurs many times in the app: 

* Ask yourself: isn't there a better way?
* Can I abstract this into a reusable function?

Reply:
Actually upon reviewing quarterUtils.js (typo in my commit), I realized that there already
exists a function that does what I want! fromFormat. No need to
implement function myself.

Reply:
Awesome!